src/locale/locales/pt-BR/messages.po

```diff
@@ -3862,7 +3862,7 @@ msgstr "Listas de Moderação"
 
 #: src/components/moderation/LabelPreference.tsx:247
 msgid "moderation settings"
-msgstr ""
+msgstr "configurações de Moderação"
```

Comment:
no need for the capital `M` in here right?

Reply:
no, In this case, everything is lowercase.